### ISSUE TITLE
Easily See the Number of Vouchers Redeemed from Admin View

### ISSUE THREAD

**Issue by Blaec-CoCalc** [sagemathinc/cocalc#7590]
This is a short issue about the possibility of including the "total number of vouchers unclaimed" (or claimed) from the initial "voucher admin view" at https://cocalc.com/vouchers/created

This would be helpful for at-a-glance follow-ups for folks who managed to claim their CoCalc Credit Voucher (and those who have not).

Currently, you have to click on the voucher to look at the listed "number of unredeemed codes" and "number of redeemed codes" shown next to "Copy Unredeemed Codes" and "Copy Redeemed Codes" respectively _in parentheses_ (see below). 

**In short, this is a non-urgent request to include the "number of unredeemed codes" (or "number of redeemed codes") as a column next to "codes" (i.e., number of codes created) at https://cocalc.com/vouchers/created.**

![Screenshot 2024-05-28 at 4 32 59 PM](https://github.com/sagemathinc/cocalc/assets/101306074/c02a84c1-ed7f-434b-b933-dcaaabe39174)
